__tests__/app.test.js

```diff
@@ -156,5 +156,66 @@ describe("news-api", () => {
                 })
             })
         })
+        describe("GET /api/articles/:article_id/comments", () => {
+            test("responds with status code 200 and an object in expected format", () => {
+                return request(app)
+                .get("/api/articles/1/comments")
+                .expect(200)
+                .then(({body}) => {
+                    expect(body).toHaveProperty("comments")
```

Comment:
This doesn't hurt, but it's probably overkill to check that the response object has a key of comments. This is checked implicitly by the fact we then check things like comments.length and whether it's an array.

Reply:
Personally I'd combine this with the test below. They both cover the format of the object.